ebsco/edsapi-koha-plugin
# 1905 additional updates

Addresses PR #74 to check for $currentRange before using it.
Addresses PR #72 and #21 to add recommended Plack compliant variable declarations.
Addresses Issues with Internal Server Errors occurring when clicking on certain catalog links and save to list links.

To Test:
Install the KPZ file on a local test instance.

Your EDS API must include your catalog, and your Koha Discovery must be configured with the proper catalog prefix and catalog ID to test the Internal server error fix.

Search for any known catalog items while in the Discovery search.
Click on the title of that item.  Note it takes you to the detailed record as expected.
Go back to the results list.
Make sure you are logged in to your account.
Add a catalog item to a list.
Note that the item adds correctly without internal server errors.
